tree/tree/inc/TLeaf.h

```diff
@@ -59,6 +59,13 @@ class TLeaf : public TNamed {
       kNewValue = BIT(12)         ///< Set if we own the value buffer and so must delete it ourselves.
    };
 
+   enum DeserializeType{
+      kInvalid = 0,      // Invalid deserialization information.
+      kDestructive = 0,  // Deserialization of this Leaf requires a separate output buffer.
```

Comment:
Why is Invalid and Destructive the same value?

Reply:
looks like a rebase issue - should be distinct.